### PR TITLE
Adds appropriate rounding to modal values

### PR DESCRIPTION
**Pull request by johnmosesman**

### CODE REVIEW

File: app/components/borrow-flow/borrow.tsx [hunk 1]
```diff
@@ -41,12 +41,10 @@ export default function Borrow({
...
     }
-    getCurrentlyBorrowing(signer, row.cToken, row.token).then(
-      (c: BigNumber) => {
-        let formattedValue: string = c.toString();
-        setCurrentlyBorrowing(formattedValue);
-      }
-    );
+    getCurrentlyBorrowing(signer, row.cToken, row.token).then((c: number) => {
+      let formattedValue: string = c.toString();
+      setCurrentlyBorrowing(formattedValue);
```
Comment (ryanlabouve) [inline app/components/borrow-flow/borrow.tsx:45]: redundant var

File: app/components/borrow-flow/borrow.tsx [hunk 2]
```diff
@@ -127,7 +125,9 @@ export default function Borrow({
               </div>
               <div className="flex items-center mb-3 text-gray-400 border-b border-b-gray-600 py-5">
                 <div className="flex-grow">Borrow Balance</div>
-                <div>${formattedBorrowedAmount}</div>
+                <div>
+                  {formattedBorrowedAmount} {row.name}
```
Comment (ryanlabouve) [inline app/components/borrow-flow/borrow.tsx:128]: You could hide this if `formattedB` is nullish

File: app/components/borrow-flow/repay.tsx [hunk 3]
```diff
@@ -135,7 +135,9 @@ export default function Repay({
           </div>
           <div className="flex items-center mb-3 text-gray-400 border-b border-b-gray-600 py-5">
             <div className="flex-grow">Borrow Balance</div>
-            <div>${formattedBorrowedAmount}</div>
+            <div>
```
Comment (ryanlabouve) [inline app/components/borrow-flow/repay.tsx:138]: You could hide this if formattedB is nullish

File: app/lib/tender.ts [hunk 4]
```diff
@@ -7,9 +7,14 @@ import SampleErc20Abi from "~/config/sample-erc20-abi";
 import SampleComptrollerAbi from "~/config/sample-comptroller-abi";
 
 import type { TokenPair } from "~/types/global";
+import { formatUnits } from "ethers/lib/utils";
 
 const MINIMUM_REQUIRED_APPROVAL_BALANCE = BigNumber.from("1");
 
+function formatBigNumber(value: BigNumber, decimals: number): number {
+  return parseFloat(parseFloat(formatUnits(value, decimals)).toFixed(2));
```
Comment (ryanlabouve) [inline app/lib/tender.ts:15]: You might decompose this into two lines or add a comment. Otherwise it hurts my brain

**Review by ryanlabouve — APPROVED** [review]
L appropriate ™

File: app/components/borrow-flow/repay.tsx (hunk 3 above)
Comment (johnmosesman) [reply app/components/borrow-flow/repay.tsx:138]: it shows 0 on the view which is probably good
image

File: app/components/borrow-flow/borrow.tsx (hunk 2 above)
Comment (johnmosesman) [reply app/components/borrow-flow/borrow.tsx:127]: see below

**Pull request merged**